Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml

```diff
@@ -0,0 +1,346 @@
+        - ChemDispenserMachineCircuitboard
+        - RiotBulletShield
+        - MakeshiftShield
+        - RiotLaserShield
+        - RiotShield
+        - TelescopicShield
+        - WebShield
+        - BedsheetUSA
+        - FlatpackerMachineCircuitboard
+        - WeaponGauntletGorilla
+        - ClothingHeadHatHetmanHat
+        - MonkeyCubeBox
+        - RCD
+        - MedkitAdvancedFilled
+        - MedkitToxinFilled
+        - MedkitRadiationFilled
+        - MedkitBurnFilled
+        - MedkitOxygenFilled
+        - MedkitFilled
+        - ResearchAndDevelopmentServerMachineCircuitboard
+        - CableApcStack
+        - BoxMRE
+        - FoodTinPeachesMaint
+        - CigCartonBlack
+        - PlushieRGBee
+        - RPED
+        - MobSlimesPet
+        - PlushieAbductor
+        - ClothingHeadAreopagite
+        - ClothingOuterHardsuitAtmos
+        - ClothingBackpackWaterTank
+        - ClothingBeltBandolier
+        - BaseBallBat
+        - ClothingOuterHardsuitBasic
+        - Bible
+        - BikeHornImplanter
+        - BoxHugHealing
+        - FoodCabbage
+        - ClothingOuterHardsuitEngineeringWhite
+        - ClothingOuterHardsuitMedical
+        - ClaymoreDulled
+        - ClothingOuterHardsuitClown
+        - WeaponShotgunDoubleBarreled
+        - ClothingHandsGlovesCombat
+        - ClothingOuterHardsuitEngineering
+        - FireAxe
+        - ClothingShoesGaloshes
+        - HandHeldMassScanner
+        - BoxPerformer
+        - HolofanProjector
+        - ProtolatheHyperConvectionMachineCircuitboard
+        - AutolatheHyperConvectionMachineCircuitboard
+        - OreProcessorIndustrialMachineCircuitboard
+        - JetpackBlueFilled
+        - KatanaDulled
+        - Kanabou
+        - LightImplanter
+        - PlushieLoveable
+        - ClothingOuterHardsuitLuxury
+        - WeaponMakeshiftLaser
+        - MedicalTrackingImplanter
+        - MedicalTechFabCircuitboard
+        - MegaSprayBottle
+        - JetpackMiniFilled
+        - ClothingOuterHardsuitSalvage
+        - WeaponPistolMk58
+        - WeaponPistolMk58Nonlethal
+        - MothroachCube
+        - PlushiePenguin
+        - PlasteelArmingSword
+        - SheetPlastic
+        - WeaponShotgunSawnNonlethal
+        - WeaponShotgunSawn
+        - ClothingOuterArmorScaf
+        - ClothingHeadHelmetScaf
+        - ScienceTechFabCircuitboard
+        - ServiceTechFabCircuitboard
+        - SheetifierMachineCircuitboard
+        - ShredderMachineCircuitboard
+        - Sledgehammer
+        - ClothingOuterHardsuitSpatio
+        - WeaponLaserSvalinn
+        - TechnologyDisk
+        - ClothingHeadHelmetTemplar
+        - SheetUranium
+        - ClothingNeckCloakVoid
+        - BoxHandcuff
+        - ResearchDisk5000
+        - ResearchDisk
+        - WeaponRevolverDeckardNonlethal
+        - ToySword
+        - ClothingShoesBootsSpeed
+        - PinpointerUniversal
+      chance: 1
+      rarePrototypes:
+        - ClothingBeltSheathFilled
```

Comment:
I dont know how it would be good for This to be remove as a chance to get yet, Sec is able to just buy it. I mean when i was sheriff the other day. all the deputys had captain swords